kolibri/plugins/coach/assets/src/csv/fields.js

```diff
@@ -0,0 +1,208 @@
+      format(row) {
+        if (!row.avgScore) {
+          return '';
+        }
+
+        return coachStrings.$tr('percentage', { value: row.avgScore });
+      },
+    },
+  ];
+}
+
+export function helpNeeded() {
+  return [
+    {
+      name: coachStrings.$tr('helpNeededLabel'),
+      key: 'helpNeeded',
+      format: row => row.total - row.correct,
+    },
+    {
+      name: coreStrings('allLabel'), // TODO: Add new string for this
+      key: 'all',
+      format: row => row.total,
+    },
+  ];
+}
+
+export function lastActivity() {
+  return [
+    {
+      name: coachStrings.$tr('lastActivityLabel'),
+      key: 'lastActivity',
+      format(row) {
+        if (!row[this.key]) {
+          return '';
+        }
+
+        return row[this.key].toISOString();
+      },
+    },
+  ];
+}
+
+export function learnerProgress(key = 'status') {
+  return [
+    {
+      name: coreStrings('progressLabel'),
+      key,
+      format(row) {
+        const [value] = at(row, key);
+        const strings = translations.learnerProgress[VERB_MAP[value]];
+        return strings.$tr('labelShort', { count: 1 });
+      },
+    },
+  ];
+}
+
+export function list(key, label) {
+  return [
+    {
+      name: coachStrings.$tr(label),
+      key,
+      format(row) {
+        const [value] = at(row, key);
+
+        if (value && value.length) {
+          return value.join(', '); // TODO: Internationalize
+        }
+
+        return '';
+      },
+    },
+  ];
+}
+
+export function name(label = 'nameLabel') {
+  return [
+    {
+      name: coachStrings.$tr(label),
+      key: 'name',
+    },
+  ];
+}
+
+export function recipients() {
+  return [
+    {
+      name: coachStrings.$tr('recipientsLabel'), // TODO: Add new string for this
+      key: 'recipientType',
+      format(row) {
+        if ('groupNames' in row && row.groupNames.length) {
+          return coachStrings.$tr('groupsLabel');
+        }
+
+        if ('hasAssignments' in row && row.hasAssignments) {
+          return coachStrings.$tr('classLabel');
+        }
+
+        return '';
+      },
+    },
+    ...list('groupNames', 'recipientsLabel'),
+  ];
+}
+
+export function score() {
+  return [
+    {
+      name: coachStrings.$tr('scoreLabel'),
+      key: 'score',
+      format: row => {
+        if (!row.statusObj.score) {
+          return '';
+        }
+
+        return coachStrings.$tr('percentage', { value: row.statusObj.score });
+      },
+    },
+  ];
+}
+
+export function tally() {
+  return [
+    {
+      name: coachStrings.$tr('startedLabel'),
+      key: 'tally.started',
+    },
+    {
+      name: coreStrings('completedLabel'),
+      key: 'tally.completed',
+    },
+    {
+      name: coachStrings.$tr('helpNeededLabel'),
+      key: 'tally.helpNeeded',
+    },
+    {
+      name: coreStrings('allLabel'), // TODO: Add new string for this
+      key: 'all',
+      format(row) {
+        return Object.values(row.tally).reduce((sum, val) => sum + val, 0);
+      },
```

Comment:
![image](https://user-images.githubusercontent.com/2367265/69767738-85db4c00-1132-11ea-91b6-65131713b9b1.png)

I think it would be more clear to list the four mutually exclusive categories: not started, started, completed, help needed.

For 0.14 I think we should update the labels to be more explicit, e.g. "Num learners started", "Num learners needing help", etc

Side note - people might see two columns with the exact same name and interpret it as a bug.

Since we can't add new strings now, recommend adding a simple number:

| Title | Average score | Recipients (1) | Recipients (2) | Not started | Started | Completed | Help needed |
|--|--|--|--|--|--|--|--|
| Quiz title | 14% | Class |   | 4 | 3| 2 | 1 |

Reply:
> I think it would be more clear to list the four mutually exclusive categories: not started, started, completed, help needed.

Agreed, except there strangely isn't a string for `Not Started` in coach or common strings so I went with `All` and a column that represents a total. The string _does_ exist in the codebase though.

> For 0.14 I think we should update the labels to be more explicit, e.g. "Num learners started", "Num learners needing help", etc

For sure. This tally format is what I was referencing in my example in https://github.com/learningequality/kolibri/issues/6151

> Side note - people might see two columns with the exact same name and interpret it as a bug.

Okay, I was worried about any string concatenation but I'll make this change.